packages/shared/src/components/cards/MarketingCTA/MarketingCTACard.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactElement } from 'react';
+import { Card } from '../Card';
+import { CardCover } from '../common/CardCover';
+import { CTAButton, Description, Header, MarketingCTA, Title } from './common';
+import { Card as CardV1 } from '../v1/Card';
+import { useFeedLayout } from '../../../hooks';
+
+export function MarketingCTACard({
+  marketingCTA,
+}: {
+  marketingCTA: MarketingCTA;
+}): ReactElement {
+  const { shouldUseMobileFeedLayout } = useFeedLayout();
+  const CardComponent = shouldUseMobileFeedLayout ? CardV1 : Card;
+
+  return (
+    <CardComponent className="p-4">
+      {marketingCTA.tagColor && marketingCTA.tagText && (
+        <Header
+          tagColor={marketingCTA.tagColor}
+          tagText={marketingCTA.tagText}
+        />
+      )}
+      <Title>{marketingCTA.title}</Title>
+      {marketingCTA.description && (
+        <Description>{marketingCTA.description}</Description>
+      )}
+      {marketingCTA.image && (
+        <CardCover
+          imageProps={{
+            loading: 'lazy',
+            alt: 'Post Cover image',
```

Comment:
Not sure if we should keep this, was afraid Marketing CTA would sound even worse

Reply:
I don't think it's useful accessibility wise as is either, this is supposed to describe the image so people using this can understand what image is shown, no need to have "image" as text inside as well.